Add threading info (#1446)
fluent/fluent-bit-docs: pipeline/inputs/network-io-metrics.md

The **netif** input plugin gathers network traffic information of the running system every certain interval of time, and reports them.

> The Network I/O Metrics plugin creates metrics that are log-based \(I.e. JSON payload\). If you are looking for Prometheus-based metrics please see the Node Exporter Metrics input plugin.

Reviewer: ```suggestion
The Network I/O Metrics plugin creates metrics that are log-based, such as JSON
payload. For Prometheus-based metrics, see the Node Exporter Metrics input plugin.
```
